kedro-org/kedro-website
# Add Heap tracking to Kedro website

Signed-off-by: Tynan DeBold <thdebold@gmail.com>

## Description

Resolves #50.

## QA notes

The Heap ID in production has `3228` as the last four digits. Check the deploy URL's browser console to confirm that.

I see it in my browser (blocked because of my extensions):

<img width="674" alt="image" src="https://user-images.githubusercontent.com/16869061/168642725-c48a1d57-ec8d-40a1-afc7-a083ac1d80fb.png">


## Checklist
